src/migrations/cypress.js

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+import { run, npm } from '../utils';
+import SDKMigration from './base';
+
+class CypressMigration extends SDKMigration {
+  static name = '@percy/cypress';
+  static version = '^3.0.0';
+
+  async upgrade() {
+    await npm.install(`${this.name}@${this.version}`);
+  }
+
+  transforms = [{
+    message: 'Percy tasks were removed, update plugins file?',
```

Comment:
`update your Cypress plugins file?`? #bikeshed

Reply:
The prompts currently read as

```
? Upgrade SDK to @percy/cypress@3.0.0?
? Percy tasks were removed, update plugins file?
? Which files? cypress/plugins/index.js
```

So I feel like adding `Cypress` to the transform message might be redundant

Reply:
Yeah, but a lot of stuff (output) happens between those two prompts. Its 2-3 gaf hahaha